test/unit/event/send-processing-ack-invalid-bank-details-error-event.test.js

```diff
@@ -0,0 +1,211 @@
+jest.mock('uuid')
```

Comment:
The most complete test file across the whole of the FFC programme -- great job 🔥